test/api_integration/apis/console/autocomplete_entities.ts

```diff
@@ -13,6 +13,149 @@ import type { FtrProviderContext } from '../../ftr_provider_context';
 export default ({ getService }: FtrProviderContext) => {
   const supertest = getService('supertest');
 
+  const createIndex = async (indexName: string) => {
+    await supertest
+      .post(`/api/console/proxy?method=PUT&path=/${indexName}`)
```

Comment:
Does it make more sense to use the `es` service for these setup/teardown utils rather than calling the Console API?

Reply:
Thanks for pointing this out! I updated the tests to use the es service instead of the Console API.